doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -469,10 +456,19 @@ arguments used for configuration.  Here is a table describing these emulated dev
 
    * - ``virtio-net``
      - Virtio network type device, parameter should be appended with the format:
-       ``virtio-net,<device_name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>]``.
-       The ``mac`` address is optional, ``device_name`` is the name of the TAP (or MacVTap) device.
-       It must include the keyword ``tap``.
+       ``virtio-net,<device_type=name>,[vhost],[mac=<XX:XX:XX:XX:XX:XX>],[mac_seed=<seed_string>]``.
+       The ``mac`` address is optional, ``name`` is the name of the TAP (or MacVTap) device.
        ``vhost`` specifies vhost backend, otherwise the VBSU backend is used.
+       ``mac_seed=<seed_string>``
+       Set a platform-unique string as a seed to generate the mac address.  Each
+       VM should have a different "seed_string".  The "seed_string" can be
+       generated by the following method where $(vm_name) contains the name of the
+       VM you are going to launch.
+
+.. code-block::
+
+   mac=$(cat /sys/class/net/e*/address)
+   seed_string=${mac:9:8}-${vm_name}
 
```

Comment:
@weichenli-intel Please indent lines 468-471 under line 466.

Reply:
I will check it.